README.md

```diff
@@ -63,13 +63,20 @@ You need to run the `renderkit` command with the following arguments as either c
 | `input`                | Template string to render                                                      | string |
 | `input-file`           | Template input file to render                                                  | string |
 | `input-dir`            | Template input directory to render                                             | string |
-| `exclude`              | Glob patterns for files to exclude from rendering                              | list   |
+| `exclude`              | Exclude files/directories using path-based glob patterns                       | list   |
 | `output`               | Output directory to write to                                                   | string |
-| `datasource`           | Datasource to use for rendering (scheme://path)                                | list   |
+| `datasource`           | Datasource to use for rendering (scheme://path) __**__                         | list   |
 | `data`                 | Data to use for rendering. Can be used to provide data directly                | list   |
 | `engine`               | Templating engine to use for rendering (Go Templates by default)               | string |
 | `allow-duplicate-keys` | Allow duplicate keys in datasources. If set, the last value found will be used | bool   |
 
+### **Notes on `datasource`
+
+- For now, only the `env` scheme is supported for datasources.
```

Comment:
Not very clear, should say that for plain files a scheme is not needed

Reply:
right!